cargo-ledger/src/main.rs

```diff
@@ -363,6 +380,46 @@ fn build_app(
         apdu_file_path.to_str().unwrap(),
     );
 
+    if standard_build_tree {
+        // Standard bin name.
+        const STANDARD_BIN_NAME: &str = "app";
+        // Closure to copy file with error message if any.
+        let copy_file_with_err_msg = |src: &Path, dst: &Path| {
+            if let Err(err) = fs::copy(src, dst) {
+                eprintln!("Error copying file to output directory : {}", err);
+            }
+        };
+        // Closure to copy generated binaries to standard path.
+        let copy_binaries_to_standard_path = |dir: &Path| {
+            let elf_dest = dir.join(format!("{}.elf", STANDARD_BIN_NAME));
+            let hex_dest = dir.join(format!("{}.hex", STANDARD_BIN_NAME));
+            let apdu_dest = dir.join(format!("{}.apdu", STANDARD_BIN_NAME));
+            println!(
+                "Copying generated binaries to standard directory : {:?}",
+                dir
+            );
+            if let Err(err) = fs::create_dir(&dir) {
+                eprintln!(
+                    "Error creating standard device build directory : {}",
+                    err
+                );
+            }
+            copy_file_with_err_msg(&exe_path, &elf_dest);
+            copy_file_with_err_msg(&hex_file_abs, &hex_dest);
+            copy_file_with_err_msg(&apdu_file_path, &apdu_dest);
+        };
+        // Copying binaries to standard bin directory.
+        let bin_dir = PathBuf::new().join(current_dir).join("bin");
+        copy_binaries_to_standard_path(&bin_dir);
```

Comment:
What is this directory ? Shall we also copy all binaries (`hex`, `apdu` and `elf`) into a `/bin` directory ?

Reply:
In c app, artifcat are both under bin/ and build/nanox/bin/